### PR TITLE
Move Compare into More dropdown

### LINKED ISSUE #440: Repo tabs doesn't fit when Wiki and Projects tabs are enabled
screen shot 2017-05-25 at 14 43 49

Maybe we should renamed `Insights` to `More` and move `Settings` there too?

### PR DESCRIPTION
**Pull request by bfred-it**
Fixes #440

Closes #501

screen shot 2017-06-19 at 11 53 37

### CODE REVIEW

File: src/libs/icons.js [hunk 1]
```diff
@@ -21,3 +21,5 @@ export const fork = '<svg aria-hidden="true" class="octicon octicon-repo-forked"
 export const cloudUpload = '<svg aria-hidden="true" class="octicon octicon-cloud-upload" height="16" role="img"  version="1.1" viewBox="0 0 16 16" width="16"><path fill-rule="evenodd" d="M7 9H5l3-3 3 3H9v5H7V9zm5-4c0-.44-.91-3-4.5-3C5.08 2 3 3.92 3 6 1.02 6 0 7.52 0 9c0 1.53 1 3 3 3h3v-1.3H3c-1.62 0-1.7-1.42-1.7-1.7 0-.17.05-1.7 1.7-1.7h1.3V6c0-1.39 1.56-2.7 3.2-2.7 2.55 0 3.13 1.55 3.2 1.8v1.2H12c.81 0 2.7.22 2.7 2.2 0 2.09-2.25 2.2-2.7 2.2h-2V12h2c2.08 0 4-1.16 4-3.5C16 6.06 14.08 5 12 5z"></path></svg>';
 
 export const compare = '<svg aria-hidden="true" class="octicon octicon-diff" height="16" version="1.1" viewBox="0 0 13 16" width="13"><path fill-rule="evenodd" d="M6 7h2v1H6v2H5V8H3V7h2V5h1v2zm-3 6h5v-1H3v1zM7.5 2L11 5.5V15c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1h6.5zM10 6L7 3H1v12h9V6zM8.5 0H3v1h5l4 4v8h1V4.5L8.5 0z"></path></svg>';
+
+export const fullCompare = '<svg xmlns="http://www.w3.org/2000/svg" class="octicon octicon-diff" height="16" viewBox="0 0 13 16" width="13"><path d="M6 7h2v1H6v2H5V8H3V7h2V5h1zm-3 6h5v-1H3zM7.5 2L11 5.5V15c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1zm1-2H3v1h5l4 4v8h1V4.5z" fill-rule="evenodd"/></svg>';
```
Comment (bfred-it) [inline src/libs/icons.js]: We don't need this one actually. This is how it'd look with fullCompare icon:

screen shot 2017-06-19 at 11 57 29

Comment (sindresorhus) [inline src/libs/icons.js]: Looks good

Comment (bfred-it) [inline src/libs/icons.js]: Which one? I ended up using the white one and removing the `fullCompare` version.

Comment (sindresorhus) [inline src/libs/icons.js]: I meant the dark one in the screenshot looks a little bit better to me as it's dark like the other icons in the dropdown.

Comment (bfred-it) [inline src/libs/icons.js]: Restored

**Pull request merged**